x/upgrade/keeper/grpc_query.go

```diff
@@ -56,7 +61,12 @@ func (k Keeper) ModuleVersions(c context.Context, req *types.QueryModuleVersions
 
 	// check if a specific module was requested
 	if len(req.ModuleName) > 0 {
-		if version, ok := k.getModuleVersion(ctx, req.ModuleName); ok {
+		version, ok, err := k.getModuleVersion(ctx, req.ModuleName)
```

Comment:
Do we really need three returned value? Cannot we make it so that if err == nil then the version is ok? And thus remove that ok bool?